docs/walkthrough/using-callbacks.md

```diff
@@ -0,0 +1,53 @@
+(using-callbacks)=
+
+# Using Callbacks
+
+Callbacks are a way of adding additional methods to the finetuning process. The methods are executed when certain events occur and there are several callback classes, each serving a different function by providing different methods for different events.
+A run can be assigned multiple callbacks using the optional `callbacks` parameter when it is created.
+
```

Comment:
add some code blocks to demonstrate how to add callbacks, for exmaple:

```python
import finetuner
from finetuner.callbacks import Callabck1, Callback2
finetuner.fit(
    callbacks=[Callback1(), Callback2(),..]
)
```

Reply:
I have added some example code now